stripe/src/main/java/com/stripe/android/view/AddAddressWidget.java

```diff
@@ -78,13 +83,53 @@ public void setOptionalFields(List<String> optionalAddressFields) {
      * @param hiddenAddressFields address fields that should be hidden. Hidden fields are
      *                            automatically optional.
      */
-    public void setHiddenFields(List<String> hiddenAddressFields) {
+    public void setHiddenFields(@Nullable List<String> hiddenAddressFields) {
+        if (hiddenAddressFields ==  null) {
+            return;
```

Comment:
If the user sets hidden address fields to null, do we want do interpret that as empty ("hide nothing")?